lib/DocumentCard/DocumentCard.js

```diff
@@ -59,13 +63,26 @@ export default class DocumentCard extends React.Component {
         <Headline data-test-doc-name={name} margin="none">
           {name}
         </Headline>
-        {note ?
-          <KeyValue label={<FormattedMessage id="stripes-erm-components.doc.note" />}>
-            <div data-test-doc-note>{note}</div>
-          </KeyValue>
-          :
-          null
-        }
+        <Row>
+          <Col xs={8}>
```

Comment:
Let's let the `Col` be the full width of 12 if there's no category. So, `xs={category ? 8 : 12}`